bgpd/bgp_mplsvpn_snmp.c

```diff
@@ -554,16 +554,23 @@ static bool is_bgp_vrf_active(struct bgp *bgp)
 {
 	struct vrf *vrf;
 	struct interface *ifp;
+	const char *vrf_name;
 
 	/* if there is one interface in the vrf which is up then it is deemed
 	 *  active
 	 */
 	vrf = vrf_lookup_by_id(bgp->vrf_id);
 	if (vrf == NULL)
 		return false;
+
+	if (bgp->name)
```

Comment:
If the default vrf is used, then bgp->name is NULL? If so, this is used in many places and could be a macro/inline function/etc.?

Reply:
I am not so sure bgp->name can be null each time default vrf is used.
I prefer to separate the problems.